### PR TITLE
Add additional points to merge process

### PR DESCRIPTION
**Pull request by Lukewh**
Following [dev catchup](https://github.com/ubuntudesign/meeting-notes/issues/325) we decided these new rules.

### CODE REVIEW

File: workflow/github.md [hunk 1]
```diff
@@ -77,6 +77,8 @@ Here are the steps that need to be completed before merging:
 * All [review labels](./labels.md#review) have to be at +1
 * Your pull request must be approved by at least one person
 * Any automated checks must have passed
+* The original author should merge a pull request
+  * This can be overuled if the author is unavailable and/or the merge is of critical importance
```
Comment (deadlight) [inline workflow/github.md]:
```suggestion
   * This can be overruled when necessary, only if the author is unavailable
```

**Comment by deadlight** [comment]
:+1: from me with the suggested change.

**Comment by nottrobin** [comment]
:+1: from me

**Comment by barrymcgee** [comment]
👍🏻

**Pull request merged**